app/components/alerts/AlertsList.js

```diff
@@ -24,15 +21,15 @@ class AlertsList extends React.Component {
 			isRefreshing: false,
 		};
 
-    this.state.getAlertsList(); //ensure newest alerts shown w/o refreshing
```

Comment:
It seems like the new alerts don't show up w/o refreshing if the user is currently in the app, which is why this is needed.

Reply:
I see, I'll have to look into that.  Thanks!